openlibrary/i18n/te/messages.po

```diff
@@ -275,78 +275,78 @@ msgstr "మీ ప్రస్తుత సంకేతపదం"
 
 #: borrow.html:85
 msgid "Loan Expires"
-msgstr ""
+msgstr "రుణం గడువు"
 
 #: borrow.html:86
 #, fuzzy
 msgid "Loan actions"
-msgstr "చర్యలు"
+msgstr " రుణం చర్యలు"
 
 #: borrow.html:114
 #, python-format
 msgid "There is one person waiting for this book."
 msgid_plural "There are %(n)s people waiting for this book."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "ఒకేఒక్క మనిషి మాత్రం ఈ పుస్తకం కోసం వేచిఉన్నాడు."
+msgstr[1] "అక్కడ %n (s ) మనుషులు ఈ పుస్తకం కోసం వేచిఉన్నారు."
```

Comment:
Where there is a `python-format` keyword,  substitutions like `%(n)s` need to appear exactly as typed in the translations.
The `s` signifies `string` rather than English plural -- it's not very clear. There are a few different formats of these python substitutions unfortunately.

(Note to self: again, something to add in a guide for translators)